tests/vmemcache_test_mt.c

```diff
@@ -194,22 +194,24 @@ init_test_get(VMEMcache *cache, unsigned n_threads, os_thread_t *threads,
 	int cache_is_full = 0;
 	vmemcache_callback_on_evict(cache, on_evict_cb, &cache_is_full);
 
-	printf("%s: filling up the pool...\n", __func__);
+	printf("%s: filling the pool...", __func__);
```

Comment:
fflush(stdout) if you really don't want \n

Reply:
Done